.github/workflows/ci.yml

```diff
@@ -5,18 +5,33 @@ on:
     branches:
       - main
       - release-*
-    tags:
-      - "v[0-9]+.[0-9]+.[0-9]+*"
   pull_request:
     branches:
       - "**"
   schedule:
     - cron: "0 4 * * *" # run every day at 4am UTC
+  workflow_call:
+    inputs:
+      is_production_release:
+        required: false
+        type: boolean
+        default: false
+      release_version:
+        required: false
+        type: string
+        default: ''
+      dry_run:
+        required: false
+        type: boolean
+        default: false
```

Comment:
Just for my own knowing, does the workflow behave as expected when `is_production_release` to `true` and `dry_run` to `true`?